functions-and-operators/string-functions.md

```diff
@@ -107,19 +107,134 @@ Return a string containing binary representation of a number.
 
 ### [`BIT_LENGTH()`](https://dev.mysql.com/doc/refman/8.0/en/string-functions.html#function_bit-length)
 
-Return length of argument in bits.
+The `BIT_LENGTH()` function is used to return the length of a given argument in bits. It calculates the length of the argument and then returns the result in bits.
```

Comment:
```suggestion
The `BIT_LENGTH()` function is used to return the length of a given argument in bits.
```

Reply:
The second sentence seems unnecessary as it expresses the same meaning as the first sentence.